tgui/packages/tgui/interfaces/Crystallizer.js

```diff
@@ -47,7 +47,7 @@ export const Crystallizer = (props, context) => {
                 width="63px"
                 unit="moles/s"
                 minValue={0}
-                maxValue={500}
+                maxValue={250}
```

Comment:
What?

Reply:
The value is currently getting halved by delta time, so I needed to halve it to properly convert it to ticks

Reply:
but its shown in seconds?

Reply:
I know, that's what I was talking about in the pr desc. That number is unlikely to be accurate with our current system, but it's a price I'm personally willing to pay

Reply:
Weird.

Reply:
Ideally this would be given through ui_static_data by the way, but oh well.